src/covid19sim/run.py

```diff
@@ -218,35 +218,41 @@ def run_simu(n_people=None,
     env = Env(start_time)
     city_x_range = (0,1000)
     city_y_range = (0,1000)
-    city = City(env, n_people, init_percent_sick, rng, city_x_range, city_y_range, start_time, Human)
-    monitors = [EventMonitor(f=1800, dest=outfile, chunk_size=out_chunk_size), SEIRMonitor(f=1440)]
+    city = City(env, n_people, init_percent_sick, rng,
+                city_x_range, city_y_range, Human)
 
-    # run the simulation
+    # Add monitors
+    monitors = [
+        EventMonitor(f=SECONDS_PER_HOUR*30, dest=outfile, chunk_size=out_chunk_size),
```

Comment:
Don't we need to divide by `TICK_MINUTE` anymore?

Reply:
See the 8th commit on the stack, `1edbae68a25ac5bf38a318118f66c963d3ca0b10`.